src/components/SearchBox.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Card, Input } from "@hackclub/design-system";
+
+const Base = Card.withComponent(Input).extend.attrs({
+    borderRadius: props => props.theme.pill,
+    boxShadowSize: "sm",
+    f: 5,
+    mt: 4,
+    px: 4
+})`
+    will-change: border-color, box-shadow;
+    &:hover {
+        box-shadow: ${props => props.theme.boxShadows[1]};
+    }
+    &:focus {
+        border-color: ${props => props.theme.colors.smoke};
+        box-shadow: ${props => props.theme.boxShadows[2]};
+    }
+`;
+
+const SearchBox = ({ onSearchChange, value }) => (
+    <Base
```

Comment:
Check this out ✨ https://github.com/kenny-hibino/react-places-autocomplete

Reply:
Looks interesting! Definitely want to add this soon. :tada: